taxi/views.py

```diff
@@ -87,3 +89,31 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    form_class = DriverLicenseUpdateForm
+    template_name = "taxi/driver_form.html"
+    queryset = Driver.objects.all()
```

Comment:
For what this queryset?

Reply:
![image](https://user-images.githubusercontent.com/110391080/220176746-dcf0330c-e513-4669-b267-4ad712e530c2.png)
without queryset